portal/common/lib/http/http_error_responses.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import template_404 from "@static/404-page.template.html";
+import template_404 from "../../static/404-page.template.html";
```

Comment:
question: I thought we wanted to use `@` imports, why the revert?

Reply:
We were using `@` imports as aliases for relative imports. 
When we started installing the common lib as a dependency, these aliases stopped working so I reverted them. 
It's fine because relative imports are only used now inside the common lib, where most files are inside the `lib/` dir, with the exception of `static/`.